### PR TITLE
Add sources for libstdc++6 >= 4.9

### PR DESCRIPTION
**Pull request by tautschnig**
clang-format-6.0 requires a libstc++6 version newer than what Ubuntu 14.04 ships
by default; such a version is available from the ubuntu-toolchain-r-test PPA.

- [x] Each commit message has a non-empty body, explaining why the change was made.
- n/a My contribution is formatted in line with CODING_STANDARD.md.
- n/a Methods or procedures I have added are documented, following the guidelines provided in CODING_STANDARD.md.
- [x] Regression or unit tests are included, or existing tests cover the modified code (in this case I have detailed which ones those are in the commit message).
- n/a My commit message includes data points confirming performance improvements (if claimed).
- [x] My PR is restricted to a single feature or bugfix.
- n/a White-space or formatting changes outside the feature-related changed lines are in commits of their own.

### CODE REVIEW

**Comment by chrisr-diffblue** [comment]
I note that the CI job for clang-format is running `git-clang-format-6.0`, but the `clang-format --version` line is reporting 5.0.0.

**Comment by tautschnig** [comment]
@chrisr-diffblue Thanks a lot, good catch! I believe to have fixed this now.

**Review by chrisr-diffblue — APPROVED** [review]
Not a code owner for this apparently (I though I was at one point) so you'll need another reviewer, but LGTM.

**Review by thk123 — APPROVED** [review]

**Review by allredj — COMMENTED** [review]
Passed Diffblue compatibility checks (cbmc commit: 5eebeafa04a78f48db5195caff74493b106b2d51).
Build URL: https://travis-ci.com/diffblue/test-gen/builds/87938871

**Pull request merged**
